src/util-mpm-hs.c

```diff
@@ -570,6 +570,134 @@ static PatternDatabase *PatternDatabaseAlloc(uint32_t pattern_cnt)
     return pd;
 }
 
+static const char *HSCacheConstructFPath(uint32_t hs_db_hash)
+{
+    static char hash_filepath[2048];
+    char hash_file_path_prefix_path[] = "/tmp/";
+    char hash_file_path_suffix[] = "_v1.hs.bin";
+    uint16_t hash_file_bytes_written = 0;
+    snprintf(hash_filepath, sizeof(hash_filepath), "%s", hash_file_path_prefix_path);
+    hash_file_bytes_written += sizeof(hash_file_path_prefix_path) - 1;
```

Comment:
this doesn't count what was written, only what it tried to write. Error checking snprinft is needed to track what was really written